internal/entities/features/features.go

```diff
@@ -120,6 +120,24 @@ var ClassFeatures = map[string][]entities.CharacterFeature{
 			Source:      "Cleric",
 		},
 	},
+	"ranger": {
+		{
+			Key:         "favored_enemy",
+			Name:        "Favored Enemy",
+			Description: "You have significant experience studying, tracking, hunting, and even talking to a certain type of enemy. Choose a type of favored enemy: aberrations, beasts, celestials, constructs, dragons, elementals, fey, fiends, giants, monstrosities, oozes, plants, or undead. Alternatively, you can select two races of humanoid (such as gnolls and orcs) as favored enemies. You have advantage on Wisdom (Survival) checks to track your favored enemies, as well as on Intelligence checks to recall information about them.",
```

Comment:
the api has all of this information. I think we just need to know what the various keys do